Modules/ITS/include/ITS/ITSTrackTask.h

```diff
@@ -118,6 +125,147 @@ class ITSTrackTask : public TaskInterface
   double ptBins[141];     // pt bins
 
   o2::itsmft::TopologyDictionary* mDict;
+
+ private:
+  //analysis for its-only residual
+  o2::its::GeometryTGeo* mGeom;
+
+  double FitStepSize0 = 0.3;
+  double FitStepSize1 = 1.0e-5;
+  double FitStepSize2 = 1.0e-5;
+  double FitStepSize3 = 1.0e-5;
+
+  double FitTolerance = 1.0e-8;
+  double ITS_AbsBz = 0.5; //T
+
+  double inputG_C[3*NLayer];
+  double FitparXY[6];
+  double FitparDZ[2];
+  ROOT::Fit::Fitter fitterA;
+  ROOT::Fit::Fitter fitterB;
+  Int_t mDefaultMomResPar = 0;
+  float mResMonTrackMinPt = 0;
+
+  TH1D* hResidualRealTimePerTrackwFinerBin;
+  TH1D* hResidualCPUTimePerTrackwFinerBin;
+  std::array<std::unique_ptr<TH2D>, NLayer> hdxySensor{};//[NLayer];
+  std::array<std::unique_ptr<TH2D>, NLayer> hdzSensor{};//[NLayer];
+
+  void circlefit_XY(double* input, double* par, double &MSEvalue, std::vector<bool> hitUpdate, int step = 0);
```

Comment:
use the camel case for all functions: `circlefit_XY` becomes `circleFitXY`. This is the convention in QC.